tests/export/html/test_markup_compatibility.py

```diff
@@ -91,3 +91,71 @@ def test_textbox_with_content_outside_of_textbox(self):
             </p>
         '''
         self.assert_document_generates_html(document, expected_html)
+
+    def test_fallback_is_only_text(self):
+        document_xml = '''
+            <p>
+                <r><t>AAA</t></r>
+                <r>
+                    <t>BBB</t>
+                    <AlternateContent>
+                        <Fallback>
+                            <r>
+                                <t>CCC</t>
+                            </r>
+                        </Fallback>
+                    </AlternateContent>
+                    <t>DDD</t>
+                </r>
+                <r><t>EEE</t></r>
+            </p>
+        '''
+
+        document = WordprocessingDocumentFactory()
+        document.add(MainDocumentPart, document_xml)
+
+        expected_html = '''
+            <p>AAABBBCCCDDDEEE</p>
+        '''
+        self.assert_document_generates_html(document, expected_html)
+
+    def test_fallback_is_a_table(self):
+        document_xml = '''
+            <p>
+                <r><t>AAA</t></r>
+                <r>
+                    <t>BBB</t>
+                    <AlternateContent>
+                        <Fallback>
+                            <tbl>
```

Comment:
To inherit the grandparents children requires that we know what the grandchild is. As such, it cannot be defined at the class level. If we are in no rush to fix this, then I am going to punt on this.